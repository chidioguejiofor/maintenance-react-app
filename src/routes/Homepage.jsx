
import React from 'react';

const Homepage = () => (
  <h1>This is my home page</h1>
);


export default Homepage;
