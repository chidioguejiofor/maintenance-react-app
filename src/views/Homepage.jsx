import React from 'react';
import Header from '../components/Header';
import Footer from '../components/Footer';


import LandingBody from '../components/LandingBody';


const buttons = [
  {
    caption: 'Register',
    className: 'primary-button',
    link: '/signup',
  },
  {
    caption: 'Login',
    className: 'primary-button',
    link: '/login',
  },
];


const Homepage = () => (
  <div id="landing-page">
    <Header />
    <LandingBody
      buttons={buttons}
    />
    <Footer />
  </div>
);


export default Homepage;
