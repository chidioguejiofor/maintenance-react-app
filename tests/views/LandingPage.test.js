/* eslint no-undef:off */
import React from 'react';
import { shallow } from 'enzyme';

import Homepage from '../../src/views/Homepage';

describe('Testing LandingPage Component', () => {
  it('should match existing snapshot', () => {
    expect(shallow(<Homepage />)).toMatchSnapshot();
  });
});
