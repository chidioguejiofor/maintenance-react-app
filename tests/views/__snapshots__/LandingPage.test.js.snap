// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing LandingPage Component should match existing snapshot 1`] = `
<div
  id="landing-page"
>
  <Header />
  <LandingBody
    buttons={
      Array [
        Object {
          "caption": "Register",
          "className": "primary-button",
          "link": "/signup",
        },
        Object {
          "caption": "Login",
          "className": "primary-button",
          "link": "/login",
        },
      ]
    }
  />
  <Footer />
</div>
`;
